x-pack/test/functional/apps/dashboard/group3/reporting/screenshots.ts

```diff
@@ -226,7 +226,7 @@ export default function ({
           updateBaselines
         );
 
-        expect(percentDiff).to.be.lessThan(0.03);
+        expect(percentDiff).to.be.lessThan(0.035);
```

Comment:
Instead of changing percentage, how about updating snapshots?

Reply:
The problem of updating the snapshot is that there will be an unknown interval between this PR is merged and the styles go to production. During that time tests would fail.

That's why I would only update the snapshot and revert this change **after** the styles are in production.